editor/src/clj/editor/game_project.clj

```diff
@@ -142,8 +142,17 @@
    ["input" "gamepads"] [[:build-targets :dep-build-targets]]
    ["input" "game_binding"] [[:build-targets :dep-build-targets]]})
 
+(defn update-build-errors [build-errors _node-id]
+  (mapv (fn [error-group]
+          (mapv (fn [error]
+                  (if (and (vector? error) (map? (first error)))
+                    [(assoc (first error) :_node-id _node-id)]
+                    error))
+                error-group))
+        build-errors))
```

Comment:
This function add `game.project` `_node-id` for all the errors to show resource in console  (we don't know `_node-id` or the `game.project` at the moment of creation these errors)
![CleanShot 2024-10-20 at 09 30 27@2x](https://github.com/user-attachments/assets/7acf05ff-442c-4d70-b962-35b56d47c2ce)

Reply:
The code of this function (very probably) can be replaced with `(some-> (g/flatten-errors build-errors) (assoc :_node-id _node-id))`. It probably should, to make the flattening more robust...